src/main/resources/messages

```diff
@@ -2,8 +2,8 @@ global.error.badRequest400.title=Bad request - 400
 global.error.badRequest400.heading=Bad request
 global.error.badRequest400.message=Please check that you have entered the correct web address.
 global.error.pageNotFound404.title=Page not found - 404
-global.error.pageNotFound404.heading=This page can’t be found
+global.error.pageNotFound404.heading=This page can''t be found
```

Comment:
One of our previous requests for accessibility was to change primes (′) to apostrophes (') for screen readers. These are the corrections, but are escaped in the messages files to allow correct output